pom.xml

```diff
@@ -4,7 +4,8 @@
 
   <groupId>tech.stackable</groupId>
   <artifactId>druid-opa-authorizer</artifactId>
-  <version>0.6.0-SNAPSHOT</version>
+  <version>23.11-SNAPSHOT</version>
```

Comment:
Must it be `23.11-SNAPSHOT` or `23.11.0-SNAPSHOT`? Omitting the patch level will prohibit us from fixing bugs and releasing them as 23.11.1 I fear.

Reply:
Good point. Will add a comment to the PR to discuss further.